integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -579,6 +583,62 @@ def save(self, *args: Any, **kwargs: Any) -> None:
             self.last_updated = timezone.now()
         super().save(*args, **kwargs)
 
+    @transaction.atomic
+    def cleanup_autosaves(self) -> None:
+        """
+        Delete all ``AUTO_SAVE``s older than the second last manual save
```

Comment:
```suggestion
        Delete all ``AUTO_SAVE`` translations older than the second last manual save
```
Haha sorry, apparently it was something else :grin:

Reply:
Thanks :pray: